doc/sphinx/source/utils/RTW/tested_recipes.rst

```diff
@@ -6,14 +6,16 @@ Currently tested recipes
 
 .. include:: common.txt
 
-The following currently tested recipes are available in the |RTW|:
+The recipes tested with the |RTW| at each site:
```

Comment:
I feel this sentence finishes prematurely. Would it be worth adding something like:
```suggestion
The recipes tested with the |RTW| at each site are defined in the following files:
```